schema/traces/README.md

```diff
@@ -0,0 +1,104 @@
+A **SpanContext** contains the tracing identifiers and the options that are propagated from parent to child Spans.
+
+* TraceId  - It is worldwide unique with practically sufficient probability by being made as 16 randomly generated bytes - used to group all spans for a specific trace together across all processes. 
+* SpanId  -  It is the identifier for a span, globally unique with practically sufficient probability by being made as 8 randomly generated bytes. When passed to a child Span this identifier becomes the parent span id for the child Span.
+* Tracestate - carries tracing-system specific context in a list of key value pairs. Tracestate allows different vendors propagate additional information and inter-operate with their legacy Id formats. For more details see this.
+
+Additional fields can be supported via the Attributes key/value store see [traces](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/README.md)
+
+### Structure
+The default fields that are supported by the traces are
+- **TraceId**  : It is worldwide unique with practically sufficient probability by being made as 16 randomly generated bytes - used to group all spans for a specific trace together across all processes.
+- **SpanId**   : It is the identifier for a span, globally unique with practically sufficient probability by being made as 8 randomly generated bytes. When passed to a child Span this identifier becomes the parent span id for the child Span.
+- **ParentId** : It is the identifier for a span's parent span.
+- **TraceState** : carries tracing-system specific context in a list of key value pairs. Tracestate allows different vendors propagate additional information and inter-operate with their legacy Id formats. For more details see this.
+
+- **Name**   :  String representing the span's name   
+- **Kind** 
+  - SpanKind.CLIENT	
+  - SpanKind.SERVER	
+  - SpanKind.CONSUMER
+  - SpanKind.PRODUCER
+  - SpanKind.INTERNAL
+
+- **StartTime** : Start time of the event
+- **EndTime**   : End time of the event
+- **Attributes**
+  - An Attribute is a key-value pair, which has the following structure  [Attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/b00980832b4b823155001df56dbf9203d4e53f98/specification/common/README.md#attribute)
+
+- **DroppedAttributesCount** : Integer counting the dropped attributes
+- **Events**     : A set of the next tuples (timestamp, name, Attributes)
+- **DroppedEventsCount** : Integer counting the dropped events
+- **Links**  : links to causally-related Spans
+- **DroppedLinksCount** : Integer counting the dropped links
+- **Status** - 
+  - UNSET= 0  -> The default status.
+  - OK= 1     -> The operation has been validated by an Application developer or Operator to have completed successfully.
+  - ERROR= 2  -> The operation contains an error.
```

Comment:
[The spec](https://github.com/open-telemetry/opentelemetry-specification/blob/b00980832b4b823155001df56dbf9203d4e53f98/specification/trace/api.md#set-status) says there should be a precedence of `Ok > Error > Unset`. The current assignment makes implementing this logic needlessly tricky (should swap ok/error), unless these values are controlled by someone else.

Reply:
I'm not sure precedence means that OK enum value has to be greater than ERROR enum value, it just means that if ok status is set >  more precedence than  > error status set  > more precedence than unset

Reply:
this is the spec in the OTEL trace protobuff:
https://github.com/open-telemetry/opentelemetry-proto/blob/e6dfd7f4cefdc2e8844fa3dfcd8b307477d59937/opentelemetry/proto/trace/v1/trace.proto#L264

Reply:
I've also added the status / message to the readme for further clarification